### PR TITLE
add knob to disable referer check

### PR DESCRIPTION
**Pull request by joecaswell**

### CODE REVIEW

**Comment by bsparrow435** [comment]
+1, should help some customers where the referrer thing gets in the way and they already have adequete security.

File: src/riak_kv_wm_utils.erl [hunk 1]
```diff
@@ -235,7 +235,8 @@ any_to_bool(V) when is_boolean(V) ->
     V.
 
 is_forbidden(RD) ->
-    is_null_origin(RD) or not is_valid_referer(RD).
+    is_null_origin(RD) or 
+    (app_helper:get_env(riak_kv,secure_referer_check,true) and not is_valid_referer(RD)).
```
Comment (reiddraper) [inline src/riak_kv_wm_utils.erl:239]: pretty sure you mean `orelse` and `andalso` here and not `or` and `and`

Comment (joecaswell) [inline src/riak_kv_wm_utils.erl:239]: You are absolutely correct.

Do you have an opinion on whether this setting should also disable the null
origin check?

Comment (reiddraper) [inline src/riak_kv_wm_utils.erl:239]: I'd lean toward yes.

**Pull request merged**